sales.html

```diff
@@ -11,23 +11,25 @@
     <div id="form">
     <form id="new-store-form">
       <fieldset>
-        <legend>Enter New Cookie Store Info</legend>
+        <legend id="formheader">Enter New Cookie Store Info</legend>
         <label for="cookieStoreName">Cookie Store Name:</label>
-        <input type="text" name="cookieStoreName" value="The Gutta">
+        <input id="storenamefield" type="text" name="cookieStoreName" value="">
```

Comment:
Technically, the for attribute in the label above looks for an id, not a name, so this should be "cookieStoreName" for that to work.